conda-env/prod.yml

```diff
@@ -0,0 +1,37 @@
+# Conda production environment for datasm operations.
+name: datasm_prod
+channels:
+  - conda-forge
+  - e3sm
```

Comment:
```suggestion
```

We don't need to reference the `e3sm` Anaconda channel anymore because all of the dependencies are available on `conda-forge` now, including `zstash`.